axiom/datasets.go

```diff
@@ -175,6 +176,16 @@ type IngestStatus struct {
 	WALLength uint32 `json:"walLength"`
 }
 
+func (s *IngestStatus) add(other *IngestStatus) {
```

Comment:
```suggestion
func (s *IngestStatus) merge(other *IngestStatus) {
```

I think `add` implies values are all added up but this is not true for the `WALLength`.